Shootingat Resolved Project/Assets/Scripts/Characters/Player/PlayerController.cs

```diff
@@ -1,13 +1,13 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 using PabloLario.Powerups;
-using PabloLario.Shooting;
+using PabloLario.Characters.Core.Shooting;
 using PabloLario.Managers;
 // using Random = UnityEngine.Random; // Un-comment in case I add using System
 
-#pragma warning disable CS0414 // Quitar miembros privados no le�dos
```

Comment:
This is a change that is coming automatically. Maybe it is because I am using Visual Studio code